packages/Panel/src/Panel.js

```diff
@@ -272,6 +279,7 @@ const defaultProps = {
   slideFrom: types.slideFroms.RIGHT,
   width: "33%",
   zIndex: zValue(7),
+  portalMount: document.body,
```

Comment:
I think this can be `null` by default.  If we restore the `<Portal>` helper component, then it will go back to deferring to `document.body` by default if no `container` is provided.